pkg/executor/kubernetes.go

```diff
@@ -658,7 +658,10 @@ func (kw *k8sWatcher) whenPodFinished(pod *v1.Pod) {
 	kw.oncePodFinished.Do(func() {
 		if pod.Status.Phase == v1.PodFailed {
 			close(kw.failed)
-			log.Debug("K8s task watcher: 'failed' channel closed")
+
+			if pod.Status.Message != "" {
+				log.Debugf("K8s task watcher: fail status message: %q", pod.Status.Message)
+			}
```

Comment:
@ppalucki - I think that we should log some information even when message is not available.